src/components/routing/routeProtectors/UserGuard.js

```diff
@@ -0,0 +1,14 @@
+import {Redirect} from "react-router-dom";
+import PropTypes from "prop-types";
+
+export const UserGuard = props => {
+    let token = localStorage.getItem("token");
+    if (token && token !== 'undefined' && token != null && token !== '') {
+        return props.children;
+    }
```

Comment:
like this we only veryfy that there is something stored in "token" but not if this token exists in our database. is this stong enough?
as it seems complicates as it would require a get.. maybe additional task?

Reply:
added new issue for that